src/EditableDialogBox/EditableItem/NumericItem.php

```diff
@@ -0,0 +1,54 @@
+<?php
+declare(strict_types=1);
+
+namespace Neusta\Pimcore\AreabrickConfigBundle\EditableDialogBox\EditableItem;
+
+use Neusta\Pimcore\AreabrickConfigBundle\EditableDialogBox\EditableItem;
+
+class NumericItem extends EditableItem
+{
+    public const ITEM_MIN_VALUE = 'minValue';
+    public const ITEM_MAX_VALUE = 'maxValue';
+    private int $min = 0;
+    private int $max = 0;
```

Comment:
We don't need default values here, they're initialized in the constructor anyway.

```suggestion

    private int $min;
    private int $max;
```